backend/services/goals_service.py

```
@@ -174,6 +182,11 @@ def populate_daily_goals(self, user_id: int):
                 .limit(1)
                 .one()
             )
+        existing_goals = UserGoal.query.filter_by(user_id=user_id).all()
+        for goal in existing_goals:
+            if goal.goal.time_period == TimePeriodType.DAILY:
+                logger.debug(f"existing goal: {goal.goal.title}, {goal.date_assigned}")
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)